storybook/storybook-react/src/Alert/Alert.docs.mdx

```diff
@@ -10,18 +10,46 @@ import README from "../../../../packages/css/src/components/alert/README.md?raw"
 
 <Controls />
 
-### Information with inline link
+### Success
+
+Laat het weten als een belangrijke actie geslaagd is.
+
+<Canvas of={AlertStories.Success} />
+
+### Warning
 
-<Canvas of={AlertStories.InformationWithInlineLink} />
+Toon een waarschuwing als er actie van de Amsterdammer nodig is.
```

Comment:
This is a bit specific, and maybe not always correct?

```suggestion
Toon een waarschuwing als er actie van de gebruiker nodig is.
```

Reply:
We’re using this in multiple places already. I think it’s a guideline, but the Schrijfwijzer is currently offline. Even though it’s not always technically correct, it’s more personal than ‘user’. We’ll be working with a microscopy specialist soon, let’s revisit later.

Reply:
We use 'gebruiker' more often than 'Amsterdammer' though, and both aren't very personal imo. I'd use 'gebruiker' for now, and maybe change it later?

Reply:
Alright, changed everything to ‘gebruiker’ for now.